### PR TITLE
Add and rename functions

### PR DESCRIPTION
**Pull request by shadowhand**
This will break BC and be released as 1.0.

### CODE REVIEW

**Comment by shadowhand** [comment]
@equip/contributors feedback?

**Comment by elazar** [comment]
@shadowhand Looks like benchmark tests need to be updated. Changes themselves look good though.

File: benchmarks/ArrayBench.php [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-namespace Equip;
+namespace Equip\Arr;
 
 class ArrayBench
 {
     public function benchHead()
     {
-        return \Equip\head($this->words());
+        return \Equip\Arr\head($this->words());
```
Comment (shadowhand) [inline benchmarks/ArrayBench.php]: @elazar updated.

**Comment by shadowhand** [comment]
@elazar should be better now.

**Comment by elazar** [comment]
👍

**Comment by shadowhand** [comment]
Version 1.0.0 has been released with these changes.

**Pull request merged**